src/PyCall.jl

```diff
@@ -28,14 +28,6 @@ if VERSION >= v"0.4.0-dev+3710"
 else
     const unsafe_convert = Base.convert
 end
-# This is not the exact version but should be closed enough
-if VERSION >= v"0.4.0-dev+4922"
```

Comment:
It would be nicer to keep this for now, as `if  v"0.4.0-dev+XXX" >  VERSION >= v"0.4.0-dev+4922"` for the correct XXXX, so it will continue to work for people who have a 0.4 snapshot in the window where `DLHandle` was used.